bin/dom_callback_null_args.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2024 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export 'package:over_react_codemod/src/executables/dom_callback_null_args.dart';
```

Comment:
I haven't reviewed this PR, but I have the same question on all these - do we want to have just one `null_safety_prep` that all these codemods can go into instead of adding a bunch of separate codemods?

Reply:
@sydneyjodon-wk I was wondering about that actually - I think thats a good idea...

Should I remove the individual `bin/` executables here and in https://github.com/Workiva/over_react_codemod/pull/273? Or can we add an aggregated executable in a follow-up PR without modifying these?

Reply:
Is there a reason we would need separate executables along with the aggregated one?

Reply:
Not necessarily - was just thinking we could create the aggregate executable in a single pr rather than having to push changes to two existing PRs

Reply:
oh yeah that works! I think we should remove the individual executables from both PRs though if we don't think they're needed if that's what you were asking!

Reply:
@sydneyjodon-wk done (in both PRs)